handlers/accesscontrol/encryption.go

```diff
@@ -0,0 +1,40 @@
+package accesscontrol
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/livepeer/catalyst-api/config"
+)
+
+type EncryptionHandlersCollection struct {
+	publicKey string
+	nodeName  string
+}
+
+func NewEncryptionHandlersCollection(cli config.Cli) *EncryptionHandlersCollection {
+	return &EncryptionHandlersCollection{
+		publicKey: cli.VodEncryptPublicKey,
+		nodeName:  cli.NodeName,
+	}
+}
+
+func (ec *EncryptionHandlersCollection) PublicKeyHandler() httprouter.Handle {
+	return func(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+
+		responseData := map[string]string{
+			"publicKey": ec.publicKey,
+			"nodeName":  ec.nodeName,
```

Comment:
I think catalyst-api uses `snake_case` so we should be consistent here
```suggestion
			"public_key": ec.publicKey,
			"node_name":  ec.nodeName,
```